ocaml/xapi/xapi_pvs_server.ml

```diff
@@ -14,13 +14,38 @@
 
 (** Implementation of PVS Server *)
 
+open Listext
+
 module D = Debug.Make(struct let name = "xapi_pvs_server" end)
 
-let not_implemented x =
-	raise (Api_errors.Server_error (Api_errors.not_implemented, [ x ]))
+let assert_ports_valid ~first_port ~last_port =
```

Comment:
This looks like a generally useful function that could go in helpers.ml alongside `assert_is_valid_ip`. Perhaps call it `assert_tcp_udp_port_range_valid`. Also `assert_port_valid` could be a top-level function in helpers.